content/vault/v1.21.x/content/docs/deploy/run-as-zcx-cluster.mdx

```diff
@@ -0,0 +1,317 @@
+---
+layout: docs
+page_title: Run Vault on IBM z/OS Container Extensions
+description: >-
+  Step-by-step guide to deploying a secure 3-node Vault cluster inside IBM zCX using HAProxy and TLS.
+---
+
+# Run Vault on IBM z/OS Container Extensions (zCX)
+
+This guide walks through deploying a fully secured 3-node HashiCorp Vault Enterprise cluster
+on IBM z/OS Container Extensions (zCX). The setup uses three independent zCX instances, 
+each running Vault with its own unique IP address. A Layer-4 HAProxy load balancer sits 
+in front to distribute traffic, and end-to-end TLS encryption is applied across all Vault nodes 
+and the load balancer to ensure secure communication throughout the cluster.
```

Comment:
```suggestion
Deploy a fully secured 3-node HashiCorp Vault Enterprise cluster on IBM z/OS
Container Extensions (zCX) with:

- Three independent zCX instances running Vault on unique IP addressed.
- A Layer-4 HAProxy load balancer to distribute traffic.
- End-to-end TLS encryption to ensure secure communication throughout the cluster.
```
Style correction: describe the outcome of following the how-to guide, not the contents of the guide